README.md

```diff
@@ -12,7 +12,10 @@ Current flow for making changes:
 - Build with `yarn build-all` to apply the changes to the .css-files
 - Create PR for review and merge to master after approval
 - Make sure you are authenticated through npm by running `npm whoami`
-- Make sure your npm user has access to publish packages to AtB's npm repo. 
-- Publish from master with `yarn pub`
+- Make sure your npm user has access to publish packages to AtB's npm repo.
+- Publish from master with `yarn pub`. When selecting version number, follow these guidelines:
+	- **Major**: Breaking change. This version require you to do code modifications. This includes cases where visual changes, like colors or illustrations, are changed in a way that could require changes.
```

Comment:
```suggestion
	- **Major**: Breaking change. This version _require_ you to do code modifications after upgrading on the consumer side. This includes cases where visual changes, like colors or illustrations, are changed in a way that could require changes.
```

Reply:
Specifying that it is from the CONSUMER is important I think. It's easy to think about versioning reflecting the changes we've made internally, but it is really just an expectation-setter for the consumer of the artifact we are creating.